app/jobs/deposit_work_job.rb

```diff
@@ -97,4 +94,36 @@ def accession?(new_cocina_object:)
     # If work already opened, then the work has changes and can be accessioned.
     deposit? && (new_cocina_object || status&.open?)
   end
+
+  def globus?
+    @globus ||= content.content_files.exists?(file_type: 'globus')
+  end
+
+  def endpoint_client_for(path)
+    GlobusClient::Endpoint.new(user_id: user.email_address, path:, notify_email: false)
+  end
+
+  def accession_or_persist_complete(new_cocina_object:, druid:)
+    if accession?(new_cocina_object:)
+      work.accession!
+      Sdr::Repository.accession(druid:)
+      # If a collection manager or reviewer has rejected a previous review, we need to approve the work again
+      work.approve! if work.rejected_review?
+    else
+      work.deposit_persist_complete!
+    end
+  end
+
+  def update_globus_content # rubocop:disable Metrics/AbcSize
+    new_endpoint_client = endpoint_client_for(GlobusSupport.new_path(user: current_user, with_uploads_directory: true))
+    work_endpoint_client = endpoint_client_for(GlobusSupport.path(work:))
+    # rename if not persisted and globus?
+    if !work_form.persisted? && new_endpoint_client.exists? && !work_endpoint_client.exists?
+      new_endpoint_client.rename(new_path: GlobusSupport.path(work:, with_uploads_directory: true))
+    end
+    # remove permissions
+    work_endpoint_client.delete_access_rule
+  rescue StandardError => e
+    raise unless e.message.include?('Access rule not found')
```

Comment:
Do we expect `Access rule not found` to happen in certain occasions? Can we document them? Do they need logging or HB alerts?

Reply:
It is actually a normal occurrence that should just be ignored. This is a terrible way to ignore it, so: https://github.com/sul-dlss/hungry-hungry-hippo/issues/1377